meteor/server/api/playout/take.ts

```diff
@@ -195,6 +195,11 @@ export function takeNextPartInnerSync(
 			isTaken: true,
 			'timings.take': now,
 			'timings.playOffset': timeOffset || 0,
+			// set transition properties to what will be used to generate timeline later:
+			'transProps.allowed': currentPartInstance ? !currentPartInstance.part.disableOutTransition : false,
```

Comment:
I think it would be good to do this set as a `transProps: literal<TransformTransitionProps>({` instead of each property individually, to help ensure any future changes to the type gets done.

It would be a good idea for the generation of this object to be pulled into a reusable function (also used in the other places we generate this type) to help ensure it is always done the same